content/en/docs/concepts/services-networking/service.md

```diff
@@ -71,14 +71,17 @@ balancer in between your application and the backend Pods.
 
 ## Defining a Service
 
-A Service in Kubernetes is a REST object, similar to a Pod.  Like all of the
-REST objects, you can `POST` a Service definition to the API server to create
-a new instance.
+A Service in Kubernetes is a REST object (just as a Pod or a ConfigMap is an object
+that you can access as a resource in the Kubernetes HTTP API.
+You can use the HTTP API to create a new Service instance, to apply changes to an
+existing Service, or to request deletion.
+
 The name of a Service object must be a valid
 [RFC 1035 label name](/docs/concepts/overview/working-with-objects/names#rfc-1035-label-names).
 
-For example, suppose you have a set of Pods where each listens on TCP port 9376
-and contains a label `app.kubernetes.io/name=MyApp`:
+For example, suppose you have a set of Pods that each listen on TCP port 9376
```

Comment:
'listens' or 'listen'?

'contains' or 'contain'?

Reply:
This is feedback on a change that's now part of PR https://github.com/kubernetes/website/pull/39499